webClient/src/app/app.component.html

```diff
@@ -43,6 +43,9 @@
       <div class="power-toggle">
         <button type="submit" class="power-toggle-button btn" [style.background-color]="powerButtonColor" [disabled]="disableButton" (click)="toggleConnection()"></button>
       </div>
+      <div>
+        <button type="submit" (click)="saveSettings()"><div class="bookmark"><i class="fa fa-bookmark"></i></div></button>
```

Comment:
Not a code problem but it seems like the placement would be better underneath the power button since it leaves empty space below. Maybe even some css like `border: none; border-radius: 2px; background: darkgrey`... just a suggestion, I'm not a designer.

Reply:
I will modify this. Thanks!